case-studies/eshoppen.md

```diff
@@ -153,6 +153,50 @@ For this component:
 * RR: A virtual machine with 2 vCPUs was used, this data was sourced from [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
 * TR: The bare metal host is split up into 16 vCPUs in total. This data was sourced from the [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
 
+
+### Front End Browser for Web application
+
+### Energy (`E`)
+
+The Quantification method used for calculating energy value is **Calculate**. We are measuring CPU utilization of the laptop client device and then using a model based on the Thermal Design Power (TDP) of the processors, number of cores etc to **estimate** the power consumption.
+
+The equation used to model the energy consumption is:
+
+P[kwH] = (Power consumed by CPU or Pc Number of cores + Power consumed by Memory or Pr + Power consumed by GPU or Pg Number of GPUs)/1000
+
+* CPU Utilization doesn’t scale linearly with power consumption, we will use the power curve as described in the [SCI Data Project “[E] Energy Estimation from Utilization Model” model](https://docs.google.com/spreadsheets/d/1Viv94rMKH-fJrfD9Nn9_qkiAg1PDfIfJUAHRX9slG7A/edit#gid=526989613)
+* TDP of laptop used (Laptop- 11th Generation Intel Core i7-1185G7 vPro /32 GB RAM) = 28 W 
+* From specs, we found that Power consumed by 4GB memory is close to 1.45 W and that by 8 GB memory is approximately 2.45 W. Also from this [article](https://medium.com/teads-engineering/estimating-aws-ec2-instances-power-consumption-c9745e347959) we can consider power consumed is approx 0.38 W/GB and for this scenario it is  close to 12.16 Watts.
+* No GPU was used hence Pg ~0
+
+
+### Carbon Intensity (`I`)
+
+* We will use regional yearly averages.
+* The region the application was run in was India.
+* We will source the Carbon Intensity from the SCI Data project and the [[I] Regional Yearly Marginal Carbon Intensity](https://docs.google.com/spreadsheets/d/1Viv94rMKH-fJrfD9Nn9_qkiAg1PDfIfJUAHRX9slG7A/edit#gid=1582216595) data set.
+
+### Embodied Carbon (`M`)
+
+The equation to calculate `M = TE * (TR/EL) * (RR/TR)`
+
+Where:
+
+* TE = Total Embodied Emissions, the sum of LCA emissions for all hardware components associated with the application server.
+* TR = Time Reserved, the length of time the hardware is reserved for use by the software.
+* EL = Expected Lifespan, the anticipated time that the equipment will be installed.
+* RR = Resources Reserved, the number of resources reserved for use by the software.
+* TR = Total Resources, the total number of resources available.
+
+For this component:
+
+* TE: We will source the embodied carbon estimates for the Dell laptop  from the [Dell site](https://i.dell.com/sites/content/corporate/corp-comm/en/Documents/dell-laptop-carbon-footprint-whitepaper.pdf) Data Set.
+* TR: 1 hr.
```

Comment:
I would use the length of time for a single browser session to checkout of eShoppen. That's more representive.